src/tt21100.toit

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2023 Toitware ApS.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the package's LICENSE file.
+
+import gpio
+import i2c
+
+TT2100_FLIP_X  := 0x1
+TT2100_FLIP_Y  := 0x2
+TT2100_FLIP_XY := 0x4
+
+INVALID_VALUE_ ::= -1
```

Comment:
The fields in the `Action` class may sometimes end up with `INVALID_VALUE_` in it. For example the `button` might have this set.

As such, we should make this variable public.
```suggestion
INVALID_VALUE ::= -1
```

Reply:
The class has `type`, so users can check type and read target fields.

Reply:
Agreed.
I would still make it public, but up to you. It's clearly not a requirement.